web_generator/test/integration/interop_gen/web_types_expected.dart

```diff
@@ -72,88 +73,217 @@ extension type HTMLTransformFunc<T extends _i2.HTMLElement,
 extension type EventManipulationFunc._(_i1.JSObject _) implements _i1.JSObject {
   external _i1.JSAny? call(_i2.Event event);
 }
+
+/// Enables basic storage and retrieval of dates and times.
 extension type Date._(_i1.JSObject _) implements _i1.JSObject {
+  /// Returns a string representation of a date. The format of the string depends on the locale.
   @_i1.JS('toString')
   external String toString$();
+
+  /// Returns a date as a string value.
   external String toDateString();
+
+  /// Returns a time as a string value.
   external String toTimeString();
+
+  /// Returns a value as a string value appropriate to the host environment's current locale.
+  /// Converts a date and time to a string by using the current or specified locale.
+  /// - [locales]:  A locale string or array of locale strings that contain one or more language or locale tags. If you include more than one locale string, list them in descending order of priority so that the first entry is the preferred locale. If you omit this parameter, the default locale of the JavaScript runtime is used.
+  /// - [options]:  An object that contains one or more properties that specify comparison options.
   external String toLocaleString();
+
+  /// Returns a date as a string value appropriate to the host environment's current locale.
+  /// Converts a date to a string by using the current or specified locale.
+  /// - [locales]:  A locale string or array of locale strings that contain one or more language or locale tags. If you include more than one locale string, list them in descending order of priority so that the first entry is the preferred locale. If you omit this parameter, the default locale of the JavaScript runtime is used.
+  /// - [options]:  An object that contains one or more properties that specify comparison options.
   external String toLocaleDateString();
+
+  /// Returns a time as a string value appropriate to the host environment's current locale.
+  /// Converts a time to a string by using the current or specified locale.
+  /// - [locales]:  A locale string or array of locale strings that contain one or more language or locale tags. If you include more than one locale string, list them in descending order of priority so that the first entry is the preferred locale. If you omit this parameter, the default locale of the JavaScript runtime is used.
```

Comment:
Can we wrap strings that are greater than the line length? See `formatter.dart` where we do this exact thing.